app/models/metrics/algorithms.rb

```diff
@@ -17,7 +17,6 @@
 #  suggested_resource_exact :integer
 #  lcsh                     :integer
 #  citation                 :integer
-#  barcode                  :integer
```

Comment:
I think this is an incorrect outcome of a rebase at some point?

Reply:
It's definitely wrong, because the migration adding `barcode` still exists in the current commit, and I see the field in `schema.db`. But, running `annotate --models` again doesn't reinsert it. I'll add it in manually, but I'm a little disconcerted by this.